### PR TITLE
Update dockerfile for multiplatform build

### PR DESCRIPTION
**Pull request by akurinnoy**
### What does this PR do?

This PR updates dockerfile in order to fix multiplatform build.

### What issues does this PR fix or reference?

N/a

### CODE REVIEW

**Comment by akurinnoy** [comment]
[dashboard-next-ci-test]

File: apache.Dockerfile [hunk 1]
```diff
@@ -16,7 +17,7 @@ RUN yarn --network-timeout 600000 && yarn install
 COPY . /dashboard/
 RUN yarn compile
 
-FROM docker.io/httpd:2.4.43-alpine
+FROM docker.io/$TARGETARCH/httpd:2.4.43-alpine
```
Comment (benoitf) [inline apache.Dockerfile:20]: why do we need targetarch ?

File: apache.Dockerfile [hunk 2]
```diff
@@ -7,7 +7,8 @@
 #
 # Contributors:
 #   Red Hat, Inc. - initial API and implementation
-FROM docker.io/node:12.20.1 as builder
+
+FROM docker.io/$TARGETARCH/node:12.20.1-buster as builder
```
Comment (benoitf) [inline apache.Dockerfile:11]: Hello,
I'm not sure to understand  why we need targetarch there and buster ?

These are multi arch images already

Comment (akurinnoy) [reply apache.Dockerfile:11]: hello @benoitf ,

if I understand it correctly, node image `node:12.20.1` doesn't support architectures `linux/s390x` and `linux/ppc64le`. But node image `node:12.20.1-buster` supports all architectures we want to target.

Another issue here that I got following error for `linux/arm64v8` when I build image from `buster` without specifying `$TARGETARCH`:
```
...
 => ERROR [linux/arm64v8 internal] load metadata for docker.io/library/node:12.20.1-buster                                                                                                     0.7s
------
 > [linux/arm64v8 internal] load metadata for docker.io/library/node:12.20.1-buster:
------
error: failed to solve: rpc error: code = Unknown desc = failed to solve with frontend dockerfile.v0: failed to create LLB definition: no match for platform in manifest sha256:5407853b6aef463dc2da182751015db96958ebe5ea6ccd3bd35fd3f77f03bf8a: not found
```

Using `$TARGETARCH` resolved this issue as well.

File: apache.Dockerfile (hunk 1 above)
Comment (akurinnoy) [reply apache.Dockerfile:20]: build fails for `linux/arm64v8` if I don't specify `$TARGETARCH`.

File: apache.Dockerfile (hunk 2 above)
Comment (benoitf) [reply apache.Dockerfile:11]: and using `docker.io:12.20.1-alpine3.12` ?

Comment (akurinnoy) [reply apache.Dockerfile:11]: let me try

**Comment by akurinnoy** [comment]
@benoitf 

without $TARGETARCH:
https://github.com/eclipse/che-dashboard/runs/1813724326?check_suite_focus=true

**Review by olexii4 — APPROVED** [review]

**Review by benoitf — APPROVED** [review]

**Review by sleshchenko — APPROVED** [review]

**Pull request merged**